### DISCUSSION TITLE
Faragher's helpful documents

### DISCUSSION THREAD

**Discussion by faragher** [markqvist/Reticulum#429] [Show and tell]
I will find a better place for this, but until then, it's better to have links available rather than forgetting if I even told anyone they exist:

**NOTICE:**

The server hosting these files suffered a massive hardware failure. They have been migrated, but I have decided to inappropriately mirror them here: https://github.com/faragher/ReticulumExamples

If the links below do not function, please check the GitHub repository.

--

**Reticulum Primer:**
https://reticulum.betweentheborders.com/primer.pdf

General Reticulum information including general familiarization, network design guidance, RNode setting specifics, and thermal and power management (including solar power)

--

**Sideband Situation Tracker:**
https://reticulum.betweentheborders.com/guidance.pdf

Designed for quick reference for a group of volunteers or first responders to get up to speed quickly for an exercise or incident.

**Comment by markqvist** [comment]
This is great work, and is already covering a lot of the stuff that currently needs covering. When you think they are ready for it, it would be great to include them in, or link to them from the main Reticulum or Sideband repositories.
